ProjectEuler/Problem2/das_p002.py

```diff
@@ -0,0 +1,19 @@
+#!/home/dasirianni/miniconda2/bin/python
```

Comment:
Yes, very few python scripts are run as `./script.py` and I would avoid unless absolutely necessary.